qa-tester/src/AdminBar.php

```diff
@@ -0,0 +1,130 @@
+<?php
+/**
+ * Class AdminBar.
+ *
+ * @package AmpProject\AmpWP_QA_Tester
+ */
+
+namespace AmpProject\AmpWP_QA_Tester;
+
+/**
+ * Class handling the plugin's admin bar menu.
+ *
+ * @since 1.0.0
+ */
+class AdminBar {
+
+	/**
+	 * Build ID of the currently installed plugin version.
+	 *
+	 * @since 1.0.0
+	 * @var string
+	 */
+	protected $build_id;
+
+	/**
+	 * Constructor.
+	 *
+	 * @since 1.0.0
+	 */
+	public function __construct() {
+		$this->build_id = get_site_option( Plugin::ID_STORAGE_KEY );
+	}
+
+	/**
+	 * Registers functionality through WordPress hooks.
+	 */
+	public function register() {
+		add_action( 'admin_bar_menu', [ $this, 'add_menu_button' ], 99 );
+		add_action( 'admin_enqueue_scripts', [ $this, 'enqueue_plugin_assets' ] );
```

Comment:
The admin bar item is not getting initialized on the frontend. I believe all that is needed is to ensure it is enqueued on the frontend:

```suggestion
		add_action( 'wp_enqueue_scripts', [ $this, 'enqueue_plugin_assets' ] );
		add_action( 'admin_enqueue_scripts', [ $this, 'enqueue_plugin_assets' ] );
```

Otherwise the admin bar shows nothing:

![image](https://user-images.githubusercontent.com/134745/85326289-e42a9c00-b481-11ea-86d8-fcba8c0c2d98.png)

DOM tree:

![image](https://user-images.githubusercontent.com/134745/85326349-fc022000-b481-11ea-9d2d-4636593137d6.png)